tests/filecheck/dialects/print/print_basics.mlir

```diff
@@ -0,0 +1,20 @@
+// RUN: xdsl-opt %s | filecheck %s
+
+builtin.module {
+    print.println "Hello world!"
+
+    %144 = "arith.constant"() {"value" = 144 : i32} : () -> i32
+    %12 = "arith.constant"() {"value" = 12 : i32} : () -> i32
+
+    print.println "Did you know that {}^2 = {}, and fib({}) = {} too!", %12 : i32, %144 : i32, %12 : i32, %144 : i32
+
+    print.println "Furthermore, {} is the smallest number with 15 divisors", %144 : i32 {"test_thing" = 32 : i32}
+}
+
+// CHECK:      builtin.module {
+// CHECK-NEXT:   print.println "Hello world!"
+// CHECK-NEXT:   %0 = "arith.constant"() {"value" = 144 : i32} : () -> i32
```

Comment:
```suggestion
// CHECK-NEXT:   %{{.*}} = "arith.constant"() {"value" = 144 : i32} : () -> i32
```

Same for all other SSAValues.

Reply:
Not sure, I think I'd like to see that I don't mess up the ordering of variables here. Switching to these regex matches doesn't really help us in any way, does it?

Reply:
It makes moving/refactor/restructuring of tests way easier as we do not really guarantee anything about SSAValue names.

I agree that we want to check that twoa re the same, but filecheck does not suppor tthat properly, so I prefer to have these regexes over having to rewrite the tests.